### ISSUE TITLE
Add the bulk get path for named ids: partial bulk validator, querier, and ops

### ISSUE THREAD

**Issue by HyeockJinKim** [lablup/backend.ai#13857]
지명된 id 들을 읽는 경로를 만들고, 전량 generic 인 어댑터 7곳을 그 위로 옮긴다. 나머지 어댑터 이전은 BA-7441 이다.

### 문제

DataLoader 가 지명한 id 들을 어댑터가 검색 액션으로 읽는다. 대표 형태 (api/adapters/object_storage/adapter.py:140):

searcher = ObjectStorageSearcher(pagination=OffsetPagination(limit=len(ids)), conditions=[ObjectStorageConditions.by_ids(ids)])
result = await processors.object_storage.global_search_object_storages.run(SearchObjectStoragesAction(searcher=searcher))
storage_map = {item.id: dto(item) for item in result.items}
return [storage_map.get(ObjectStorageID(i)) for i in ids]

호출 경로는 GraphQL 리졸버 -> DataLoader(load_fn) (api/gql/data_loader/data_loaders.py) -> adapter.batch_load_\*(ids) -> Search\*Action 이다.

어긋나는 것 둘.

- 권한 확인이 지명된 개체 각각이 아니라 scope 하나에 걸린다. global search 면 SUPERADMIN 게이트다.
- 없는 id 와 권한 없는 id 가 둘 다 None 이라 구분되지 않는다.

그리고 pagination=len(ids) + by_ids + map + 순서 재배열 보일러플레이트가 어댑터 전체 53곳에 반복된다.

### 할 일 1 - 경로 신설

- 부분 응답 bulk 검증자. 거부된 id 를 예외로 올리지 않고 결과에 실패로 싣는다. 권한 확인은 이미 개체별 매핑을 받아오며 지금은 그것을 전부 아니면 전무로 접는다.
- BulkEntityQuerier - 행 자신의 id 로만 키를 잡는 읽기 spec. 조건·정렬·페이지네이션이 없으므로 spec 이 질의 전체를 소유한다.
- ops partial_bulk_get - Mapping[id, Data] 와 실패 목록.
- ProcessorGroup.bulk_get_ops - entity 대상. FieldGroup 에는 bulk_get_ops 가 이미 있으나 ProcessorGroup 에는 없다.

### 할 일 2 - 적용 대상 7곳

배선이 전량 generic 이라 깨끗하게 옮겨지는 어댑터다. 하나씩이 간단하므로 한두 개만 실험하지 않고 일곱을 다 옮겨 형태를 확인한다.

| 어댑터 | 배선 | batch_load |
|---|---|---|
| object_storage | 6/6 generic | 1 |
| runtime_variant | 6/6 | 1 |
| app_config_fragment | 7/7 | 1 |
| storage_namespace | 5/5 | 1 |
| app_config_allow_list | 5/5 | 1 |
| prometheus_query_preset_category | 4/4 | 1 |
| app_config_definition | 4/4 | 1 |

app_config_fragment 는 entity 가 아니라 field type 이다. FieldGroup.bulk_get_ops 가 이미 있으므로 그쪽 경로로 간다. entity 경로와 field 경로가 둘 다 성립하는지 확인하는 자리이므로 빼지 말 것.

서로 다른 concern 이 섞여 있다 (artifact_registry / system / app_config / metric). 한 concern 에만 맞는 형태가 나오는 것을 막는다.

### 확인할 것

옮기면 data_loaders.py 의 load_fn 이 받는 값이 달라진다. 지금은 없는 id 도 권한 없는 id 도 전부 None 인데, 새 경로는 둘을 나눠 돌려준다. DataLoader 가 그것을 어떻게 표현할지 정하고, 일곱 곳에 같은 방식으로 적용한다.

### 범위 밖

나머지 어댑터 46곳은 BA-7441 이 도메인 묶음으로 가져간다. rbac(9) deployment(7) scheduling_history(4) session·resource_group·image·vfolder 등은 generic 비율이 낮거나 다른 이슈가 도는 중이라 이 이슈에서 건드리지 않는다.

소유자 id 로 다른 테이블의 지정된 행을 읽는 것은 bulk lookup 계열이며 이 이슈 밖이다.

JIRA Issue: BA-7414
